gui/default/syncthing/core/syncthingController.js

```diff
@@ -2777,6 +2777,12 @@ angular.module('syncthing.core')
             $scope.advancedConfig = angular.copy($scope.config);
             $scope.advancedConfig.devices.sort(deviceCompare);
             $scope.advancedConfig.folders.sort(folderCompare);
+            $scope.advancedConfig.defaults.ignores._lines = function (newValue) {
+                if (arguments.length) {
```

Comment:
`newValue.length`?

Reply:
Doesn't work when the argument is `undefined` (omitted), which happens when it's called as a getter function. The technique is taken straight from from Angular's documentation.

Reply:
TIL that there's an `arguments` object in all JS functions.